build/target.cmake

```diff
@@ -1,14 +1,121 @@
+    endif()
+    SET (WALRUS_CXXFLAGS_DEBUG -O0 -Wall -Wextra -Werror)
+    SET (WALRUS_CXXFLAGS_RELEASE -O2 -fno-stack-protector -fno-omit-frame-pointer)
+    SET (WALRUS_THIRDPARTY_CFLAGS -w -g3 -fdata-sections -ffunction-sections -fno-omit-frame-pointer -fvisibility=hidden)
+    SET (WALRUS_DEFINITIONS ${WALRUS_DEFINITIONS} "WASM_API_EXTERN=__attribute__((visibility(\"default\")))")
+
+    IF (WALRUS_SMALL_CONFIG)
+        SET (WALRUS_CXXFLAGS_RELEASE ${WALRUS_CXXFLAGS_RELEASE} -Os)
+    ENDIF()
+    IF (WALRUS_DEBUG_INFO)
+        SET (WALRUS_CXXFLAGS ${WALRUS_CXXFLAGS} -g3)
+    ENDIF()
+    SET (WALRUS_CXXFLAGS_SHAREDLIB -fPIC)
+    SET (WALRUS_LDFLAGS_SHAREDLIB -ldl)
+    SET (WALRUS_CXXFLAGS_STATICLIB -fPIC -DWALRUS_EXPORT=)
+    SET (WALRUS_CXXFLAGS_SHELL -DWALRUS_EXPORT= -frtti -std=c++17)
+
+ELSEIF ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") # Clang and AppleClang
+    SET (WALRUS_CXXFLAGS
+        ${WALRUS_CXXFLAGS}
+        -std=c++11 -g3
+        -fno-rtti
+        -fno-math-errno
+        -fdata-sections -ffunction-sections
+        -fno-omit-frame-pointer
+        -fvisibility=hidden
+        -fno-fast-math -fno-unsafe-math-optimizations -fdenormal-fp-math=ieee
+        -Wno-type-limits -Wno-unused-result -Wno-unused-variable -Wno-invalid-offsetof -Wno-unused-function
+        -Wno-deprecated-declarations -Wno-unsupported-floating-point-opt -Wno-parentheses-equality -Wno-dynamic-class-memaccess -Wno-deprecated-register
+        -Wno-expansion-to-defined -Wno-return-type -Wno-overloaded-virtual -Wno-unused-private-field -Wno-deprecated-copy -Wno-atomic-alignment
+        -Wno-ambiguous-reversed-operator -Wno-deprecated-enum-enum-conversion -Wno-deprecated-enum-float-conversion -Wno-braced-scalar-init -Wno-unused-parameter
+    )
+    SET (WALRUS_CXXFLAGS_DEBUG -O0 -Wall -Wextra -Werror)
+    SET (WALRUS_CXXFLAGS_RELEASE -O2 -fno-stack-protector -fno-omit-frame-pointer)
+    SET (WALRUS_THIRDPARTY_CFLAGS -w -g3 -fdata-sections -ffunction-sections -fno-omit-frame-pointer -fvisibility=hidden)
+    SET (WALRUS_DEFINITIONS ${WALRUS_DEFINITIONS} "WASM_API_EXTERN=__attribute__((visibility(\"default\")))")
+
+    IF (WALRUS_SMALL_CONFIG)
+        SET (WALRUS_CXXFLAGS_RELEASE ${WALRUS_CXXFLAGS_RELEASE} -Os)
+    ENDIF()
+    IF (WALRUS_DEBUG_INFO)
+        SET (WALRUS_CXXFLAGS ${WALRUS_CXXFLAGS} -g3)
+    ENDIF()
+    SET (WALRUS_CXXFLAGS_SHAREDLIB -fPIC)
+    SET (WALRUS_LDFLAGS_SHAREDLIB -ldl)
+    SET (WALRUS_CXXFLAGS_STATICLIB -fPIC -DWALRUS_EXPORT=)
+    SET (WALRUS_CXXFLAGS_SHELL -DWALRUS_EXPORT= -frtti -std=c++17)
+ELSE()
+    MESSAGE (FATAL_ERROR ${CMAKE_CXX_COMPILER_ID} " is Unsupported Compiler")
+ENDIF()
+
+# Default options per host
 IF (${WALRUS_HOST} STREQUAL "linux")
+    FIND_PACKAGE (PkgConfig REQUIRED)
+    IF (WALRUS_THREADING)
```

Comment:
What is this `WALRUS_THREADING` option for?
It seems that this option is nowhere used now

Reply:
I removed that option